examples/ecs/system_closures.rs

```diff
@@ -0,0 +1,28 @@
+use bevy::prelude::*;
+
+fn main() {
+    App::build()
+        .add_system(Box::new(move |mut cmd: Commands, arg: Local<String>| {
+            info!("this system uses an argument: {:?}", arg);
+            let id = cmd.spawn().id();
+            info!("also it spawned an entity {:?}", id);
+        }).system().config(|config| config.1 = Some("hello".to_string())))
```

Comment:
```suggestion
        }).system())
```
Unless the `.config()` is completely necessary here, I feel like it's just adding unnecessary noise.

Reply:
s tI think it is necessary -- that's what sets the `Local<String>`.